packages/flutter_tools/test/commands.shard/hermetic/create_usage_test.dart

```diff
@@ -237,12 +237,11 @@ void main() {
           'create',
           '--no-pub',
           '--template=plugin',
-          '--ios-language=objc',
           'testy',
         ]);
         expect(
           (await command.unifiedAnalyticsUsageValues('create')).eventData['createIosLanguage'],
```

Comment:
Delete all usages of `createIosLanguage` in this PR, including this entire test. There is no reason to track that event any more since this PR will make 100% of the calls "swift"`.